webapp/src/Twig/TwigExtension.php

```diff
@@ -1080,7 +1080,7 @@ public function problemBadge(ContestProblem $problem): string
         // Pick the foreground text color based on the background color.
         $foreground = ($background[0] + $background[1] + $background[2] > 450) ? '#000000' : '#ffffff';
         return sprintf(
-            '<span class="badge problem-badge" style="background-color: %s; min-width: 28px; border: 1px solid %s"><span style="color: %s;">%s</span></span>',
+            '<span class="badge problem-badge" style="background-color: %s; min-width: 37px; border: 1px solid %s"><span style="color: %s;">%s</span></span>',
```

Comment:
```suggestion
            '<span class="badge problem-badge" style="background-color: %s; min-width: 3em; border: 1px solid %s"><span style="color: %s;">%s</span></span>',
```
I did not check what would be the right value but this is what I meant.

Reply:
I played with it and 2.3 seems ok